app.js

```diff
@@ -81,6 +95,19 @@ function displayTodos() {
         todoList.append(todoEl);
 
         // > Part C: Add a click event listener for the todoEl
+        todoEl.addEventListener('click', async () => {
+            const response = await completeTodo(todo.id);
+            error = response.error;
+            const updatedTodo = response.data;
+
+            if (error) {
+                displayError();
+            } else {
+                const index = todos.indexOf(todo);
+                todos[index] = updatedTodo;
+                displayTodos;
```

Comment:
`displayTodos` is missing `()` at the end of it in order for it to call the function. once you fix this is should show the css for completed todos!

Reply:
fixed it! i wondered why i had to refresh to show the lines through.....